env-example

```diff
@@ -0,0 +1,2 @@
+SECRET_KEY='django-insecure-c2k2w&t@d+6&2!)_9pnc396@wj3fns_08lz0q50o7&llp4ful4'
```

Comment:
tutaj dajcie jakąś wartość jak np. `change_me`, czy `your_secret_key_goes_
here`, żeby było wiadomo, że wiecie, że to tylko przykład